### DISCUSSION TITLE
What extensions should be built-in on Automatic?

### DISCUSSION THREAD

**Discussion by mark-wd** [vladmandic/sdnext#985] [General]
Following from #982

Automatic comes with pre-installed extensions that might or might not be useful to the end-user. While they can be disabled, they cannot really be uninstalled, as every update puts them back into place.

In particular:
- steps-animation
- seed_travel
- aesthetic-scorer

Seem to be very user-specific and not fit for a broader use case, being probably better suited as user-picked extensions.

Things like:
- system-info
- rembg
- Lycoris
- Controlnet

All seem to be direct extensions of the features of Automatic and normal image workflow, and should probablt remain built-in.

**Comment by mihaiiancu** [comment]
I like the ability to use two upscalers in extras
also the ability to remove background in extras - had a bitch of a time with that extension, until I fixed it

**Comment by vladmandic** [comment]
First three are included only because this is where they were developed to start with, before repo became a thing. I have no problem moving them to external location.

In addition, I think I'll have to create a sort of extension blacklist as there are couple of popular extensions that are known to cause problems and I'm getting multiple daily tickets for their issues.

**Comment by mark-wd** [comment]
Sounds like a plan, Vlad.

Maybe when you implement a blacklist, make a ticket in their repo about the issues that are being reported here... just to avoid some drama.

**Comment by vladmandic** [comment]
Yeah, I agree.
But ideally, I'd really need some help to do all that.

**Comment by mark-wd** [comment]
Unfortunately I didn't find the time yet to learn the elements that make up the SD enviornment and don't have prior experience with gradio and such, but if there are non-coding tasks on the table, let me know.

**Reply by vladmandic** [reply]
there is tons that you can do to help even without coding! ping me on discord if you really want to help? vlado#8083
